test/functional/mining_prioritisetransaction.py

```
@@ -110,8 +153,12 @@ def run_test(self):
         assert_raises_rpc_error(-1, "prioritisetransaction", self.nodes[0].prioritisetransaction)
         assert_raises_rpc_error(-1, "prioritisetransaction", self.nodes[0].prioritisetransaction, '')
 
-        # Test `prioritisetransaction` invalid extra parameters
-        assert_raises_rpc_error(-1, "prioritisetransaction", self.nodes[0].prioritisetransaction, '', 0, 0)
+        # Test `prioritisetransaction` invalid extra parameters (Dash uses 2-parameter format)
+        assert_raises_rpc_error(-8, "txid must be of length 64", self.nodes[0].prioritisetransaction, '', 0)
+
+        # Test `getprioritisedtransactions` invalid parameters
+        assert_raises_rpc_error(-1, "getprioritisedtransactions",
+                self.nodes[0].getprioritisedtransactions, True)
```

Ruff (0.12.2)

Boolean positional value in function call

(FBT003)